OpenRA.Mods.Common/Traits/SpawnActorsOnSell.cs

```diff
@@ -70,7 +70,11 @@ void Emit(Actor self)
 			var buildingInfo = self.Info.TraitInfoOrDefault<BuildingInfo>();
 
 			var eligibleLocations = buildingInfo != null ? buildingInfo.Tiles(self.Location).ToList() : new List<CPos>();
-			var actorTypes = info.ActorTypes.Select(a => new { Name = a, Cost = self.World.Map.Rules.Actors[a].TraitInfo<ValuedInfo>().Cost }).ToList();
+			var actorTypes = info.ActorTypes.Select(a => new
+			{
```

Comment:
I think our style convention is that if something like a var takes more than 1 line, we tab-indent lines 2+ (in this case line 74-77) to make it more visible at first glance that this is part of the var.

Reply:
This is something we are inconsistent about across the code base.  My preference is with what @IceReaper has done here.